src/Components/NotFound/NotFound.js

```diff
@@ -6,8 +6,8 @@ const NotFound = () => {
     <section className='notfound-container'>
       <h1 className='notfound-header'>Oh no! This page either moved or doesn't exist. Try something else.</h1>
       <div className='notfound-blurb'>
-        <p>PS - Hope you're doing well. 🤍</p>
-        <p>PPS - Proud of you.</p>
+        <p>P.S. - Hope you're doing well. 🤍</p>
+        <p>P.P.S. - Proud of you.</p>
```

Comment:
nice attn to detail

Reply:
Between this and "disguisted" I feel attacked

Reply:
Just looked at this for so long trying to see what the difference was. Chrissy your attention to detail always amazes me.